x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/detail_panel/detail_panel.js

```diff
@@ -502,6 +483,7 @@ export class DetailPanelUi extends Component {
         aria-labelledby="followerIndexDetailsFlyoutTitle"
         size="m"
         maxWidth={600}
+        style={{ zIndex: 5999 }} // Prevent context menu popover appearing above confirmation modal
```

Comment:
Can we create an SCSS class and use the `$euiZMask` variable instead? This will future-proof the fix against changes to styles in EUI.

```scss
@import '~@elastic/eui/src/global_styling/variables/index';

.ccrFollowerIndexDetailPanel {
  z-index: $euiZMask - 1;
}
```

Reply:
great idea! will revise